inductiva/_cli/cmd_quotas/list.py

```diff
@@ -0,0 +1,56 @@
+"""List the user quotas information via CLI."""
+from typing import TextIO
+import argparse
+import sys
+
+from inductiva import users, _cli
+from inductiva.utils import format_utils
+
+
+def get_quotas(_, fout: TextIO = sys.stdout):
+    """ Lists the user's quotas. 
+
+    Lists all the user's quotas and the quotas left for the user to use.
+    """
+    quotas = users.get_quotas()
+
+    name_list = list(quotas)
+    max_allowed_list = [value["max_allowed"] for value in quotas.values()]
+    in_use_list = [value["in_use"] for value in quotas.values()]
+
+    # Construct the dictionary of lists
+    quotas_dict_of_lists = {
+        "name": name_list,
+        "max_allowed": max_allowed_list,
+        "in_use": in_use_list
+    }
```

Comment:
whuy not use a defaultdict (`from collections import defaultdict`) to avoid multiple iterations over the dict?

```suggestion
    table = defaultdict(list)
    for name, quota in users.get_quotas().items():
        table["name"].append(name)
        table["in_use"].append(quota["in_use"])
        table["max_allowed"].append(quota["max_allowed"])
```